pkgs/os-specific/linux/sgx/ssl/default.nix

```diff
@@ -1,44 +1,39 @@
 { stdenv
 , fetchFromGitHub
-, fetchpatch
 , fetchurl
 , lib
+, openssl
 , perl
 , sgx-sdk
 , which
 , debug ? false
 }:
 let
   sgxVersion = sgx-sdk.versionTag;
-  opensslVersion = "1.1.1l";
+  opensslVersion = "1.1.1u";
 in
-stdenv.mkDerivation rec {
+stdenv.mkDerivation {
   pname = "sgx-ssl" + lib.optionalString debug "-debug";
   version = "${sgxVersion}_${opensslVersion}";
 
   src = fetchFromGitHub {
     owner = "intel";
     repo = "intel-sgx-ssl";
     rev = "lin_${sgxVersion}_${opensslVersion}";
```

Comment:
They also have a [`3.0_Rev2`](https://github.com/intel/intel-sgx-ssl/tree/3.0_Rev2) version that builds against OpenSSL 3.0.

I'm not sure what the implications are for users of sgx-ssl, but given that OpenSSL 1.1 is EOL, it might be good to get rid of it here as well, if possible.

This can be another PR though.

Reply:
Good catch! I'll take a look at this in a follow-up PR 👍